src/components/molecules/ContentCard.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+	<BaseCard v-bind="$props">
+		<div class="content-card">
+			<template v:slot:content>
+				<div class="content">
+					<BaseLink :href="url" target="_blank">
```

Comment:
```suggestion
					<BaseLink :href="url" target="_blank" :no-style="true">
```
add the `noStyle`-prop here that I explained above to remove the underline